### PR TITLE
Latest & greatest version of Stapler

### PR DESCRIPTION
**Pull request by kohsuke**

### CODE REVIEW

**Comment by kohsuke** [comment]
Why is it picking up bits from `http://repo.cloudbees.com/content/repositories/central`? It should be looking at `https://repo.jenkins-ci.org/public`

**Comment by batmat** [comment]
Nice open/close christmas tree ;-).
NIT: maybe adding stapler in the commit title would improve a slightly little wee bit that PR.

**Comment by kohsuke** [comment]
At least one of the failures look legit. I need to look at this.

**Comment by kohsuke** [comment]
Free +1 opportunities for eveyrone,  @reviewbybees

**Comment by andresrc** [comment]
:bee:

**Comment by stephenc** [comment]
:bee:

File: test/src/test/java/hudson/model/ManagementLinkTest.java [hunk 1]
```diff
@@ -47,11 +48,15 @@
      */
     @Test
     public void links() throws Exception {
-        HtmlPage page = j.createWebClient().goTo("manage");
-        List<?> anchors = DomNodeUtil.selectNodes(page, "id('management-links')//*[@class='link']/a[not(@onclick)]");
-        assertTrue(anchors.size()>=8);
-        for(HtmlAnchor e : (List<HtmlAnchor>) anchors) {
-            e.click();
+        WebClient wc = j.createWebClient();
```
Comment (jglick) [inline test/src/test/java/hudson/model/ManagementLinkTest.java]: How does this relate to a new version of Stapler?

**Pull request merged**